src/logging.cc

```diff
@@ -428,6 +438,13 @@ namespace  {
 #endif
 
 namespace {
+typedef struct filetime {
+  std::string name;
+  time_t time;
+  bool operator<(const struct filetime& o) const {
+    return o.time > time;
+  }
+} Filetime;
```

Comment:
```suggestion
struct Filetime {
  std::string name;
  std::time_t time;
  bool operator<(const struct Filetime& o) const {
    return time < o.time;
  }
};
```
It's also not clear what `time` refers to. Is it the creation, access, or modification time? Is `name` the full path? Brief comments above each member are sufficient.

Reply:
done